docs/advanced.md

```diff
@@ -221,6 +221,28 @@ with httpx.Client(headers=headers) as client:
     ...
 ```
 
+## Monitoring download progress
+
+If you need to monitor download progress of large responses, you can use stream and the `response.last_raw_chunk_size` property.
+
+For example, showing a progress bar using the [`tqdm`](https://github.com/tqdm/tqdm) library while a response is being downloaded could be done like this…
+
+```python
+import tempfile
+
+import httpx
+from tqdm import tqdm
+
+with tempfile.NamedTemporaryFile() as download_file:
+    data = b"@" * 1000000
+    with httpx.stream("POST", "https://httpbin.org/anything", data=data) as response:
```

Comment:
I think the example given is a bit misleading, since it's the *download* that we're tracking the progress of here, not the upload. Perhaps just a GET of a largish file would be a little more clear?...

```python
url = "http://ipv4.download.thinkbroadband.com/100MB.zip"
with httpx.stream("GET", url) as response:
    ...
```

Reply:
Good point,

I changed the examples (but I used the test file from this page https://speed.hetzner.de/)